ex2_system/source/housekeeping/housekeeping_task.c

```diff
@@ -36,12 +36,12 @@ SAT_returnState start_housekeeping_daemon(void);
  */
 static void * housekeeping_daemon(void *pvParameters) {
     TickType_t hk_delay = pdMS_TO_TICKS(1000);

```

Comment:
One of us should make all the system tasks delays static global and create getter/setters so we can change the delays from GS or other tasks for example. But it's another pull request's problem.

Reply:
I agree. Outside the scope of what I'm doing right now. but still an important task

Reply:
@wagnerd97 Can you add a task in clickup to add that functionality so that someone gets to it?